docs/TESTS.md

```diff
@@ -59,8 +62,7 @@ public class ExampleTest {
         totalCounter = targetInstance.getAtomicLong("totalCounter");
         counter = targetInstance.getAtomicLong("counter");
 
-        selector.addOperation(Operation.PUT, putProbability)
-                .addOperation(Operation.GET, getProbability);
+        operationSelectorBuilder.addOperation(Operation.PUT, putPercentage).addOperation(Operation.GET, getPercentage);
```

Comment:
In the future (not now) we should be able to let the test framework take care of this. It can read the properties and build up the selector builder.